okrs/2019-q3.md

```diff
@@ -0,0 +1,27 @@
+## Objective: Projects that depend on IPLD are using new interfaces and specifications
+
+* Complete unixfsv2 implementation and relevant `iq` functionality (@mikeal)
+* Document `iq` and write a blog post (@mikeal)
+* Complete selector work necessary for GraphSync (@warpfork)
+* TODO: go-ipld-prime OKR's
+* TODO: schema OKR's
+
+## Objective: Building applications on IPLD is simple and fun
+
+* Build a delicio.us clone w/ IPLD (@mikeal)
+* Complete STAC geo data IPLD demo (@vmx)
+* TODO: what sample applications would @rvagg and @warpfork like to build?
+ 
+## Objective: Realize the vision of decentralized linkable advanced data structures
+
+* WASM
+  * Step 1: Implement `dag-cbor` codec in Rust (@vmx)
+    * Side Goal: Begin the work of creating a good user developer experience for IPLD in Rust.
+  * Step 2: Compile `dag-cbor` from Rust to WASM and create a working JS library on top. (@vmx)
+  * Step 3: Create an JS/WASM interface for IPLD Generic Operations (@vmx)
+  * Step 4: Port IPLD Generic implementations to Rust/WASM (@vmx)
```

Comment:
Those steps sound good. I guess for Q3 I'd pick steps 1-3, where I think 1-2 will be completed and 3 is the stretch goal (given that with vacations and conferences I have about 4 weeks less.